src/guide/reactivity-computed-watchers.md

```diff
@@ -205,6 +205,24 @@ watch([fooRef, barRef], ([foo, bar], [prevFoo, prevBar]) => {
 })
 ```
 
+### Watching Reactive Objects
+
+Using a watcher to compare values of an Array or Object that are reactive requires that it has a copy made of just the values.
+
+```js
+const numbers = reactive([1,2,3,4])
+
+watch(
+  () => [...numbers],
+  (numbers, prevNumbers) =>{
+    console.log(numbers, prevNumbers);
+})
+
+numbers.push(5) // logs: [1,2,3,4,5] [1,2,3,4]
+```
+
+For deeply nested object and arrays, a deep copy of values may be required. This can be achieved with a utility such as `lodash.cloneDeep` 
```

Comment:
**suggestion**: as we're assuming some non-Vue knowledge here, it would be nice to add a link:
```suggestion
For deeply nested object and arrays, a deep copy of values may be required. This can be achieved with a utility such as [lodash.cloneDeep](https://lodash.com/docs/4.17.15#cloneDeep)
```